### ISSUE TITLE
tvOS and watchOS support

### ISSUE THREAD

**Issue by asdfugil** [utmapp/UTM#2594]
support for running UTM on tvOS/watchOS

(OK maybe not watchOS, that seems somewhat ridiculous but tvOS makes sense)

**Comment by brunocastello** [comment]
The AppleTV does not have more than 3GB of RAM, so I do not see how it would make sense.

**Comment by asdfugil** [comment]
Debian LXDE works fine with 768 MB of ram, as well as some other Linux distributions.

**Comment by asdfugil** [comment]
also note that a jailbreak is absolutely required as normal apps can only access 500 KB of persistent local storage on Apple TV.

**Comment by brunocastello** [comment]
But it's a pretty limited configuration in terms of memory and disk storage. The only things you can run are classic DOS, Windows 3.11, macOS 9, and that's it. An iPad with 3GB of RAM struggles to run a linux without desktop GUI for web development. I can imagine that the Apple TV will be worse.

Maybe tvOS support should only be considered if Apple updates the device with the M1 and more RAM. But I do not see them doing that because tvOS was never meant to be more than just a TV streaming and light gaming device.

**Comment by Crabor** [comment]
I have an idea, if I can run UTM on tvOS, then I can run Home assistant, then I can control all non-Homekit devices through siri. This is really great! And the power consumption is extremely low!

**Comment by trinityhades** [comment]
> Since I can run Docker on a router with 1GB of RAM and 256MB of flash memory, why can't I do the same on an Apple TV?

I agree @Crabor, I went ahead and began creating a tvOS supported version. It still has a lot broken, but I was able to successfully boot into ReactOS on my Apple TV 4K Gen 1

You can use my fork: https://github.com/trinityhades/UTM
